### PR TITLE
Network CLI Commands: ARM RouteTables

### PR DESCRIPTION
**Pull request by ruslangabitov**
Changes:
    Added support for route tables

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@ruslangabitov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: lib/commands/arm/network/routeTable._js [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+      }
+    });
+  },
+
+  showRoute: function (resourceGroupName, routeTableName, routeName, options, _) {
+    var self = this;
+    var route = self.getRoute(resourceGroupName, routeTableName, routeName, _);
+
+    self.interaction.formatOutput(route, function (route) {
+      if (route === null) {
```
Comment (yugangw-msft) [inline lib/commands/arm/network/routeTable._js]: will using "!route" just work? You other code in the same file is using that already, unless here you do want to differentiate 'null, 'undefined', '[]' ...'

**Comment by yugangw-msft** [comment]
please fix the indent in the test files. The other looks good.

**Comment by babumuralidharan** [comment]
@yugangw-msft review comments updated and I guess we should be good here!!

File: test/util/networkTestUtil.js [hunk 2]
```diff
@@ -57,7 +57,38 @@ networkTestUtil.prototype.deleteUsedGroup = function (groupName, suite, callback
 			});
 	} else callback();
 };
-
+networkTestUtil.prototype.createRouteTable = function (groupName, RouteTablePrefix, location, suite, callback) {
+	var cmd = util.format('network route-table create -g %s -n %s -l %s --json', groupName, RouteTablePrefix, location).split(' ');
```
Comment (yugangw-msft) [inline test/util/networkTestUtil.js]: this indent is still too wide

**Pull request merged**